src/components/messageBus.ts

```diff
@@ -112,6 +112,13 @@ export type Command =
 			engine: Engine;
 			storageUri: Uri;
 			uri: Uri;
+	  }>
+	| Readonly<{
+			kind: 'executeCodemod';
```

Comment:
is that supported by the nora node engine?

Reply:
not yet,  that is what we need to add to the engine